attributes/default.rb

```diff
@@ -255,3 +255,6 @@
 default['datadog']['legacy_integrations']['nagios']['enabled'] = false
 default['datadog']['legacy_integrations']['nagios']['description'] = 'Nagios integration'
 default['datadog']['legacy_integrations']['nagios']['config']['nagios_log'] = '/var/log/nagios3/nagios.log'
+
+# For custom gem servers on restricted networks
```

Comment:
`clear_sources` is only available since chef `12.3` (see https://docs.chef.io/resource_chef_gem.html). Could you mention here that this attribute can be used only on Chef >= 12.3? (on lower versions the recipe won't compile)